### PR TITLE
ipycanvas Notebook

### LINKED ISSUE #126: Add ipycanvas notebook

### PR DESCRIPTION
**Pull request by martinRenou**
Fix #126

### CODE REVIEW

**Comment by mwcraig** [comment]
Please update the dependencies for the tutorial if necessary

**Comment by martinRenou** [comment]
Yes! Will do

**Comment by marimeireles** [comment]
I did it for ipygany!

**Comment by martinRenou** [comment]
@mwcraig it should be good now, thanks for the review!

**Comment by mwcraig** [comment]
Looks great, merging!

File: notebooks/07.01-ipycanvas.ipynb [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+   "metadata": {},
+   "source": [
+    "<!--NAVIGATION-->\n",
+    "< [More widget libraries](07.00-More_widget_libraries.ipynb) | [Contents](00.00-index.ipynb) | [pythreejs: 3D rendering in the browser](07.02-pythreejs.ipynb) >"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
```
Comment (mwcraig) [inline notebooks/07.01-ipycanvas.ipynb:528]:
```suggestion
   "display_name": "widgets-tutorial",
```

File: notebooks/07.01-ipycanvas.ipynb [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+   "language": "python",
+   "name": "python3"
```
Comment (mwcraig) [inline notebooks/07.01-ipycanvas.ipynb:530]:
```suggestion
   "name": "widgets-tutorial"
```

**Pull request merged**